src/main/webapp/app/entities/modeling-assessment/uml-element.model.ts

```diff
@@ -0,0 +1,25 @@
+import {ModelElementType} from "app/entities/modeling-assessment/modeling-assessment.model";
+
+export interface UMLElement {
```

Comment:
can you have a look at Apollon? I think they define the same types there and it would be better to import their definitions, so that we don't need to redefine them. In addition: in case Apollon changes something, we only find the mismatch during runtime and not during compile time

Reply:
apollon uses a different data representation then the server does such that they are not compatible. The added interfaces are supposed to represent the server view